### ISSUE TITLE
Incorrect unicode handling when copying tweets to clipboard

### ISSUE THREAD

**Issue by jscholes** [manuelcortez/TWBlue#150]
### Steps to reproduce

Locate a tweet containing emojis or other non-ASCII characters and copy it to the clipboard.

### Expected result

The tweet is copied exactly as it was posted, including all non-ASCII characters.

### Actual result

The tweet is unintentionally converted to ASCII, undoubtedly due to Python 2's insane string handling, and all non-ASCII characters are replaced with a questionmark.

**Comment by codeofdusk** [comment]
Should be resolved once the PY3 branch is merged into mainline.

Bill
